op-e2e/e2eutils/setup.go

```diff
@@ -317,4 +315,71 @@ func RedeployValPoolToTerminate(
 	return deployTx, upgradeTx, nil
 }
 
+func parseSegLenToBigInts(s string) ([]*big.Int, error) {
+	s = strings.Trim(s, "[]")
```

Comment:
As far as I know, we receive input as comma-separated values without brackets. Do we need to trim brackets?